app/views/incomes/nextYear/UpdateIncomeCYPlus1Template.scala.html

```diff
@@ -51,7 +52,8 @@
     title = heading,
     authedUser = Some(user),
     backLinkContent = Some(Messages("tai.back-link.upper")),
-    backLinkId = "backLink"
+    backLinkId = "backLink",
+    formForErrorSummary = formForErrorSummary
```

Comment:
if you look on line 27 there is already a form passed into the template, so we can delete line 30 and on line 56 we can do `formForErrorSummary = form`. Then there is no need to pass this https://github.com/hmrc/tai-frontend/pull/1032/files#diff-9e4208f0234cd30a1902e9ae11999863a25e778bd4f2a69ff9f53bc2184bed9fR39

ps - I done the exact same thing :)